tests/ctst/Dockerfile

```
@@ -13,6 +13,7 @@ COPY ./steps /ctst/steps
 COPY ./world /ctst/world
 
 USER root
+RUN npm install typescript -g
```

best to specify the version of typescript, for reproducible builds

```suggestion
RUN npm install typescript@5.8.3 -g
```